web_services/foxx/brightid/index.js

```diff
@@ -194,156 +196,91 @@ const handlers = {
     });
   },
 
-  allVerificationsGet: function(req, res){
+  verificationPublicGet: function(req, res){
     const appKey = req.param('app');
-    const count_only = 'count_only' in req.queryParams;
     const app = db.getApp(appKey);
-    const context = db.getContext(app.context);
-    const coll = arango._collection(context.collection);
-    let contextIds = db.getLastContextIds(coll, app._key);
-    let data = {
-      count: contextIds.length
-    }
-    if (! count_only){
-      data['contextIds'] = contextIds
+    const roundedTimestamp = req.param('roundedTimestamp');
+    const verification = req.param('verification') || app.verification;
+
+    const tp = app.timestampPrecision;
+    const serverRoundedTimestamp = tp ? parseInt(Date.now() / tp) * tp : 0;
+    if (serverRoundedTimestamp != roundedTimestamp) {
+      throw new errors.InvalidRoundedTimestampError(serverRoundedTimestamp, roundedTimestamp);
     }
```

Comment:
Wrap this is in an `if` block that checks to see if they're using timestamp-based verifications.